### PR TITLE
Manual Linter changes

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Blockwise manual changes
- UI/UX controls manual changes

### Types of changes

- Improvement

### How has this been tested?

- Drag and drop each block and setting check.
- Each control work fine or not

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/cf7-designer/render.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const Render = ( props ) => {
 		html = formJson.data.html;
 	}
 
-	if ( formId == 0 ) {
+	if ( formId == 0 ) { // eslint-disable-line eqeqeq
```
Comment (sandeshjangam) [inline src/blocks/cf7-designer/render.js:54]: Convert to int and then compare do not ignore

File: Gruntfile.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ module.exports = function ( grunt ) {
 					potFilename: 'languages/ultimate-addons-for-gutenberg.pot',
 					exclude: [ 'admin/bsf-core' ],
 					potHeaders: {
-						'poedit': true,
+						poedit: true,
```
Comment (RajkiranBagal) [inline Gruntfile.js]: Single quotes Removed, Is this expected?

File: src/blocks/cf7-designer/edit.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import styling from './styling';
 import React, { lazy, useEffect, Suspense } from 'react';
 import lazyLoader from '@Controls/lazy-loader';
-import { __ } from '@wordpress/i18n';
+import jQuery from 'jquery';
```
Comment (RajkiranBagal) [inline src/blocks/cf7-designer/edit.js:4]: Why imported jQuery here?

File: src/blocks/faq/styling.js [hunk 4]
```diff
@@ -92,10 +92,10 @@ function styling( props ) {
 	let iconColorTemp = iconColor;
 	let iconActiveColorTemp = iconActiveColor;
 
-	if ( 'undefined' === typeof iconColor || '' == iconColor ) {
+	if ( 'undefined' === typeof iconColor ) {
```
Comment (RajkiranBagal) [inline src/blocks/faq/styling.js:95]: Do we not need the empty condition here?

File: src/blocks/gf-designer/edit.js [hunk 5]
```diff
@@ -3,14 +3,14 @@ import React, { lazy, Suspense, useEffect } from 'react';
 import lazyLoader from '@Controls/lazy-loader';
 import { __ } from '@wordpress/i18n';
 import { SelectControl, Placeholder } from '@wordpress/components';
+import jQuery from 'jquery';
```
Comment (RajkiranBagal) [inline src/blocks/gf-designer/edit.js:6]: Why do we need to import jQuery here?

File: src/blocks/how-to/render.js [hunk 6]
```diff
@@ -434,10 +434,6 @@ const Render = ( props ) => {
 												);
 											} }
 											className="uagb-tools__label"
-											placeholder={ __(
-												'Description',
```
Comment (RajkiranBagal) [inline src/blocks/how-to/render.js:438]: Do we not need placeholder here?

File: src/components/typography/range-typography.js [hunk 7]
```diff
@@ -54,7 +54,7 @@ export default function RangeTypographyControl( props ) {
 		<>
 			<Range
 				{ ...props }
-				label={ __( props.sizeMobileText ) }
+				label={ __( props.sizeMobileText ) } // eslint-disable-line  @wordpress/i18n-no-variables
```
Comment (sandeshjangam) [inline src/components/typography/range-typography.js:57]: It is not translatable text. remove translation function.

File: src/components/typography/range-typography.js [hunk 8]
```diff
@@ -38,7 +38,7 @@ export default function RangeTypographyControl( props ) {
 		<>
 			<Range
 				{ ...props }
-				label={ __( props.sizeTabletText ) }
+				label={ __( props.sizeTabletText ) } // eslint-disable-line  @wordpress/i18n-no-variables
```
Comment (sandeshjangam) [inline src/components/typography/range-typography.js:41]:
```suggestion
				label={ props.sizeTabletText }
```

File: src/components/typography/range-typography.js (hunk 7 above)
Comment (sandeshjangam) [reply src/components/typography/range-typography.js:57]: Do not ignore rule here.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/typography/font-typography.js [hunk 9]
```diff
@@ -128,14 +128,14 @@ function FontFamilyControl( props ) {
...
+		const { loadGoogleFonts, fontFamily, fontWeight, fontSubset } = props; // eslint-disable-line no-shadow
 		props.setAttributes( { [ fontFamily.label ]: value } );
 		onLoadGoogleFonts( loadGoogleFonts, value );
 		onFontChange( fontWeight, fontSubset, value );
 	};
 
-	const onFontChange = ( fontWeight, fontSubset, fontFamily ) => {
-		let font_flag;
+	const onFontChange = ( fontWeight, fontSubset, fontFamily ) => { // eslint-disable-line no-shadow
+		let font_flag;  // eslint-disable-line no-unused-vars
```
Comment (sandeshjangam) [inline src/components/typography/font-typography.js:138]: We have this vaibale but it not used anywhere. Why we need it?

File: src/blocks/timeline/post-timeline/components/Excerpt.js [hunk 10]
```diff
@@ -4,7 +4,7 @@ const Excerpt = ( props ) => {
 
 	if ( attributes.displayPostExcerpt && undefined !== post.uagb_excerpt ) {
 		if ( attributes.displayPostExcerpt && post.uagb_excerpt ) {
-			var trimmed_excerpt = post.uagb_excerpt
+			var trimmed_excerpt = post.uagb_excerpt // eslint-disable-line no-var
```
Comment (sandeshjangam) [inline src/blocks/timeline/post-timeline/components/Excerpt.js:7]: Define let variable and then use it. do not ignore the rule

File: src/blocks/timeline/day-align-classes.js [hunk 11]
```diff
@@ -5,12 +5,12 @@
...
-	if ( 'left' == attributes.timelinAlignment ) {
+	if ( 'left' === attributes.timelinAlignment ) {
 		dayAlignClass = 'uagb-timeline__day-new uagb-timeline__day-left';
-	} else if ( 'right' == attributes.timelinAlignment ) {
+	} else if ( 'right' === attributes.timelinAlignment ) {
 		dayAlignClass = 'uagb-timeline__day-new uagb-timeline__day-right';
-	} else if ( 'center' == attributes.timelinAlignment ) {
-		if ( index_val % 2 == '0' ) {
+	} else if ( 'center' === attributes.timelinAlignment ) {
+		if ( index_val % 2 == '0' ) { // eslint-disable-line eqeqeq
```
Comment (sandeshjangam) [inline src/blocks/timeline/day-align-classes.js:13]: Can we do something like this?
```suggestion
		if ( index_val % 2 === 0 ) {
```

File: src/blocks/timeline/content-timeline/edit.js [hunk 12]
```diff
@@ -243,7 +243,7 @@ const ContentTimelineComponent = ( props ) => {
 					lineInner.height(
 						viewportHeightHalf - Math.abs( photoViewportOffsetTop )
 					);
-					++num;
+					++num; // eslint-disable-line no-unused-vars
```
Comment (sandeshjangam) [inline src/blocks/timeline/content-timeline/edit.js:246]: Do we really need this var?

File: src/components/typography/range-typography.js (hunk 7 above)
Comment (AbhijitNage123) [reply src/components/typography/range-typography.js]: Fixed.

File: src/blocks/cf7-designer/render.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/cf7-designer/render.js]: Updated

File: src/components/typography/range-typography.js (hunk 8 above)
Comment (AbhijitNage123) [reply src/components/typography/range-typography.js]: Updated

File: Gruntfile.js (hunk 2 above)
Comment (AbhijitNage123) [reply Gruntfile.js]: No, it was due to js lint we have fixed that now. we have added a rule for it.

File: src/blocks/faq/styling.js (hunk 4 above)
Comment (AbhijitNage123) [reply src/blocks/faq/styling.js:95]: No, we don't, `iconColor` never getting empty. It is always getting `undefined`. So we don't need that condition anymore. I have already tested the same by consoling `iconColor` and removing color, resetting it and on first drag and drop.

File: src/blocks/timeline/post-timeline/components/Excerpt.js (hunk 10 above)
Comment (AbhijitNage123) [reply src/blocks/timeline/post-timeline/components/Excerpt.js]: Updated

**Pull request merged**